Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts

```diff
@@ -370,19 +373,12 @@ module VORLON {
         private _generateColorfullLink(link: HTMLAnchorElement, receivedObject: any): void {
             this._appendSpan(link, "nodeName", receivedObject.name);
             
-            if (receivedObject.id) {
-                this._appendSpan(link, "nodeAttribute", " id");
-                this._appendSpan(link, "nodeTag", "=\"");
-                this._appendSpan(link, "nodeValue", receivedObject.id);
-                this._appendSpan(link, "nodeTag", "\"");
-            }
-
-            if (receivedObject.classes) {
-                this._appendSpan(link, "nodeAttribute", " class");
-                this._appendSpan(link, "nodeTag", "=\"");
-                this._appendSpan(link, "nodeValue", receivedObject.classes);
-                this._appendSpan(link, "nodeTag", "\"");
-            }
+            receivedObject.attributes.forEach(function(attr){
+                var node = document.createElement('span');
+                node.className = 'nodeAttribute';
+                node.innerHTML = '<span>' + attr[0] + '</span><span>' + attr[1] + '</span>'; 
```

Comment:
Yes, this definitely prints the ="" for all names and values. Can you/do you want to pull in an take a look in a separate branch? Or do you want me to add a screenshot to this issue thread?